code/modules/hydroponics/grown.dm

```diff
@@ -43,9 +43,6 @@
 	/// Should we pixel offset ourselves at init? for mapping
 	var/offset_at_init = TRUE
 
-/obj/item/food/grown/New(loc, obj/item/seeds/new_seed)
```

Comment:
We did this for a reason see #78414. @vinylspiders thoughts on is this still nessassary?